src/request.py

```diff
@@ -1,46 +1,56 @@
-import json
 import logging
+from typing import Dict, List
 from xml.etree import ElementTree
-from src.logger import init_logging
 
 import requests
 
+from src.logger import init_logging
+
 log = logging.getLogger()
 
 
-def request(method_type: str, url: str, **kwargs):
-    log.debug(f"started request to url '{url}'")
-    params = None
-    data = None
-    if method_type == "GET":
-        params = kwargs.get("payload", None)
-    else:
-        data = kwargs.get("payload", None)
-    response = requests.request(method=method_type,
-                                url=url,
-                                params=params,
-                                data=data,
-                                auth=kwargs.get("auth", None))
-    if response.status_code != 200:
-        log.exception(f'response status code !=200 (response status code {response.status_code})')
-        raise ValueError("response status code !=200 ", response.text)
-    try:  # try if response xml, in other case consider that it is json or bytes
+class Request:
+    @staticmethod
+    def _request_basic(method_type: str, url: str, **kwargs) -> requests.Response:
+        log.debug(f"started requester to url '{url}'")
+        params = None
+        data = None
```

Comment:
You can drop “if” and default None values, just always do both .get’s

Reply:
done